UMAProject/Assets/UMA/Core/Extensions/DynamicCharacterSystem/Scripts/UMAAssetIndexer.cs

```diff
@@ -94,29 +170,65 @@ public static void StopTimer(System.Diagnostics.Stopwatch st, string Status)
 #endif
         }
 
-        public static UMAAssetIndexer Instance
+#if DBLOGGER
+		public static void DBLogger(string message)
+		{
+			Debug.Log(message);
+			if (conn == null)
+			{
+				string connstr = "server=mcserver;user=admin;database=AddressableData;port=3306;password=f2672b555bd2fc54c985";
+				conn = new MySqlConnection(connstr);
+			}
+			if (conn == null)
+			{
+				Debug.Log("Unable to log to DB");
+				return;
+			}
+			string sql = "insert values here)";
+			MySqlCommand cmd = new MySqlCommand(sql, conn);
+			cmd.ExecuteNonQuery();
+		}
```

Comment:
Shouldn't this be handled with an interface rather than pasting mysql stuff in here?

Reply:
Removed DBLogger - this was hardcoded to a machine in my living room. :)